src/main/java/uk/gov/beis/els/categories/televisions/model/TelevisionsForm.java

```diff
@@ -1,102 +1,87 @@
 package uk.gov.beis.els.categories.televisions.model;
```

Comment:
PR already done for this file in [ELG-6](https://github.com/UKGovernmentBEIS/energy-label-service/pull/87)